segregation/tests/test_absolute_clustering.py

```diff
@@ -11,7 +11,7 @@ def test_Absolute_Clustering(self):
         s_map = s_map.to_crs(s_map.estimate_utm_crs())
         df = s_map[['geometry', 'HISP', 'TOT_POP']]
         index = AbsoluteClustering(df, 'HISP', 'TOT_POP')
-        np.testing.assert_almost_equal(index.statistic, 0.060976687541645404,  decimal=4)
+        np.testing.assert_almost_equal(index.statistic, 0.060701540314293256,  decimal=4)
```

Comment:
In the rest of the library, we generally use `numpy.testing.allclose` using the tolerances in `libpysal.common.RTOL/libpysal.common.ATOL`. Fine to use something different though if it's appropriate here! Just thought I'd note.

Reply:
ah, will check it out